discord/api/dataConverters.py

```diff
@@ -49,5 +49,5 @@ def convert_guild_member_update(self, data):
     def convert(self, event, data):
         func: typing.Callable = self.converters.get(event)
         if not func:
-            raise NotImplementedError(f"No converter has been implemented for {event}")
+            return data
```

Comment:
this would need to be `return [data]`, as this would otherwise conflict with how arguments are parsed for events